portfolio/components/ui/Figures/ReceiptHealthExplorer/ReceiptHealthExplorer.module.css

```
@@ -0,0 +1,1668 @@
+  opacity: 1;
+}
+
+.validationStatusIcon {
+  justify-self: end;
+  width: 18px;
+  height: 18px;
+  overflow: visible;
+}
+
+.validationStatusIcon circle {
+  fill: currentColor;
```

Stylelint (17.13.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)